### ISSUE TITLE
Clarify the 2nd note

### ISSUE THREAD

**Issue by clatimier** [AdobeDocs/experience-manager-cloud-service.en#162]
The 2nd note in the page mentions "If custom code was pushed to staging and then rejected by you, the next AEM update will remove those changes to reflect the git tag of the last successful customer release to production."

In case some changes have been pushed to the Stage environment for testing purpose, they will indeed be removed by the next deployment even if the customer doesn't explicitly reject them. 
Could we rephrase this to something clearer like for example
"If custom code was pushed to staging and not to production, the next AEM update will remove those changes to reflect the git tag of the last successful customer release to production. Hence the custom code that was only available on staging will have to be deployed again."

[Logged By support]

Issue in ./help/implementing/deploying/aem-version-updates.md

**Comment by bohnertchris** [comment]
@adobe export issue to Jira project CQDOC

**Comment by bohnertchris** [comment]
Thank you for the suggestion, @clatimier! We will review.
